operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -48,7 +48,7 @@ kiali_defaults:
         use_kiali_token: false
         username: ""
       display_link: true
-      in_cluster_url: "http://grafana.istio-system:3000"
+      in_cluster_url: ""
```

Comment:
Why didn't you write as you suggested in my PR : `http://grafana.{{ kiali_vars.istio-namespace }}:3000` ? It seems better to me ; so that we can have an empty in_cluster_url if it's what we want